packages/driver/src/cypress.ts

```diff
@@ -403,15 +403,9 @@ class $Cypress {
         break
 
       case 'runner:end':
-        // mocha runner has finished running the tests
+        $scriptUtils.destroySourceMaps()
```

Comment:
Could we call `$sourceMapUtils.destroySourceMapConsumers()` directly, and avoid the indirection of adding this to $scriptUtils?